testbed/tests/widgets/test_table.py

```diff
@@ -304,6 +330,59 @@ async def test_multiselect(
         assert len(multiselect_widget.selection) == 100
 
 
+async def test_multiselect_keyboard_control(
+    multiselect_keyboard_widget,
+    multiselect_keyboard_probe,
+    source,
+    on_select_handler,
+):
+    """A multiselect table can be controlled by keyboard.
+    This is a workaround to create a scenario where multiple items are
+    selected while invoking VirtualItemSelectionRangeChanged event.
```

Comment:
We generally don't mention specific platform details here; this is a generic testbed used by all platforms, not just Windows.